index/src/index.rs

```diff
@@ -1,36 +1,52 @@
 use crate::LittIndexError::{
-    CreationError, OpenError, PdfNotFoundError, PdfParseError, WriteError,
+    CreationError, OpenError, PdfNotFoundError, PdfParseError, ReloadError, WriteError,
 };
 use crate::Result;
 use litt_shared::search_schema::SearchSchema;
 use lopdf::Document as PdfDocument;
+use std::fs::create_dir_all;
+use std::path::{Path, PathBuf};
+use tantivy::query::QueryParser;
 use tantivy::schema::{Document as TantivyDocument, Schema};
-use tantivy::{Index as TantivyIndex, IndexWriter};
+use tantivy::{Index as TantivyIndex, IndexReader, IndexWriter, ReloadPolicy, Searcher};
 use walkdir::{DirEntry, WalkDir};
 
+const INDEX_DIRECTORY_NAME: &str = ".litt-index";
```

Comment:
I somehow have an aversion to hyphens :D 

I thus thought that maybe it is better to use `.litt/index`. This would also add the possibility to create other directories in case they are needed in the future.